### ISSUE TITLE
svtyper on tumor/normal pair, which BAM to supply

### ISSUE THREAD

**Issue by fpbarthel** [hall-lab/svtyper#86]
Hi, I am using `svtyper` on a VCF file produced using `lumpyexpress` given both a tumor and a matched normal sample. I thus have:
1. One "somatic" lumpy VCF file
2. Two BAM files for the tumor and normal sample

Do I:
1. Run `svtyper` pointing `-B` to the tumor BAM
2. Run `svtyper` pointing `-B` to the normal BAM
3. Both 1 and 2

**Comment by fpbarthel** [comment]
This took me some time to figure out, but option (3) is actually not possible using `svtyper-sso`.

```
barthf$ svtyper --help
...
svtyper
description: Compute genotype of structural variants based on breakpoint depth
...
  -B FILE, --bam FILE   BAM or CRAM file(s), comma-separated if genotyping multiple samples
```

I guess the "**SINGLE sample**" should give it away? It does mention you can supply multiple BAMs using comma, but this feature doesn't seem to work. I tried and it returns errors.

```
barthf$ svtyper-sso --help
...
svtyper
description: Compute genotype of structural variants based on breakpoint depth on a SINGLE sample
...
  -B FILE, --bam FILE   BAM or CRAM file(s), comma-separated if genotyping multiple samples
```

**Comment by ernfrid** [comment]
You are correct about sso, the command-line documentation is in error there. That executable will only work with single samples. You can try using svtyper (classic) with comma separated BAMs or run each BAM separately on the same input VCF then merge together with something like `svtools vcfpaste` or the system-level `paste` cmd on Unix-like systems.
